### PR TITLE
Add base .travis.yml from Twenty Sixteen

### PR DESCRIPTION
**Pull request by josephfusco**
WordPress Username: [joefusco](https://profiles.wordpress.org/joefusco)

### CODE REVIEW

**Comment by josephfusco** [comment]
Resolves #29 by introducing the travis config used for Twenty Sixteen development.

File: .travis.yml
Comment (ntwb) [inline .travis.yml]: This should be "Twenty Seventeen"

File: .travis.yml
Comment (ntwb) [inline .travis.yml:86]: There is no `.jscsrc` file currently in this pull request, either remove lines #69-#71 and #85-#86 or include the `.jscsrc` file from the Twenty Sixteen repo https://github.com/WordPress/twentysixteen/blob/master/.jscsrc

**Review by ntwb — CHANGES_REQUESTED** [review]

**Review by grappler — APPROVED** [review]

**Review by ntwb — APPROVED** [review]
Thanks @josephfusco, looks good 👍

File: .jscsrc
Comment (ernilambar) [inline .jscsrc]: May be `assets/js/html5.js` ?

File: .travis.yml
Comment (ernilambar) [inline .travis.yml]: Current development version should be 4.7

**Review by ntwb — APPROVED** [review]

**Comment by josephfusco** [comment]
**Reminder**: once merged twentyseventeen will need to be added to https://travis-ci.org/wordpress/

**Comment by josephfusco** [comment]
This would be really helpful to implement ASAP with the recent surge in issues and PRs.

**Comment by ernilambar** [comment]
I am also not sure why this is on hold. This should have been in high priority in the review.

**Pull request merged**
